content/tutorials/publish-file-links.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Publish assets using hosting services
+description: Tutorial about using hosting services like Google drive, IPFS, or AWS for publishing assets.
+---
+
+## Overview
+
+While publishing assets on Ocean Marketplace, publishers must provide a link (a URL) to the asset from where the content of the asset can be accessed. The URL of the asset is encrypted and is stored as a part of DDO on the blockchain. When a buyer pays for the asset, **Provider** decrypts this content and acts as a proxy to serve the asset. It is up to the asset publisher to decide where to host the asset. For example, a publisher can store the content on their google drive, AWS server, private cloud server, or other third-party hosting services. The DDO only stores the location of the asset, which is accessed on-demand by the Provider. 
+
+On Ocean Marketplace, a publisher must provide the link to the asset during publish step. Once the asset is published, this link cannot be changed. So, it is essential that the publisher correctly sets this field (shown in the below image).
+
+![Publish - File URL field](./images/marketplace/publish/marketplace-publish-file-field.png)
+
+## Using hosting services
+
+Publishers can choose any hosting service of their choice. The below section explains how to use commonly used hosting services with Ocean Marketplace.
+
+### Google drive
+
+Google Drive allows users to share files/folders with various access policies. Publishers must set the access policy such that anyone with the link can download the file when using Ocean Marketplace with Ocean Protocol's default Provider (https://v4.provider.rinkeby.oceanprotocol.com).
+
+#### Step 1: Get link
```

Comment:
Can we keep the same steps formatting as we have for the other tutorials?
Step 1 - Get link. Without ":" but with " - ".
Also, can we add the steps to the top page content links?

Reply:
TOC depth is a global setting for the whole docs project. [here](https://github.com/oceanprotocol/docs/blob/36fa3a4fa3d18437768dcfe977ebcd6cb572703a/src/templates/Doc.jsx#L121). If we increase the depth, it might affect other content as well.